_posts/2018-02-07-cockpit-161.md

```diff
@@ -0,0 +1,86 @@
+---
+title: Cockpit 161
+author: pitti
+date: 2018-02-07 10:00
+tags: cockpit linux
+slug: cockpit-161
+category: release
+summary: Cockpit with Virtual Machines, LVM, and Hardware Information improvements
+comments: 'true'
+---
+
+Cockpit is the [modern Linux admin interface](http://cockpit-project.org/). We release regularly.
+Here are the release notes from version 161.
+
+### New VMs can be created on Machines page
+
+The Machines page now has a "Create New VM" button for installing a new libvirt
+virtual machine from a local or remote ISO image. It allows the user to set
+some basic properties such as memory and storage sizes.
+
+![Machines Create new VM](/images/machines-create-iso.png)
+
+Thanks to Dominik Perpeet and suomiy for this feature!
+
+### VMs running in Kubernetes can now be deleted
+
+[Cockpit 160](http://cockpit-project.org/blog/cockpit-160.html) added an
+initial overview of kubevirt Virtual Machines. This page now got a first action
+to delete a running VM.
+
+![kubevirt Delete](/images/kubernetes-kubevirt-delete.png)
+
+Thanks to Marek Libra for this feature!
+
+### Improve LVM volume resizing
+
+Cockpit now knows upfront whether shrinking or growing an LVM logical volume is
+possible, and whether the filesystem needs to be unmounted.  Since shrinking
+and growing are quite different in practice, the Storage page now shows
+separate "Shrink" and "Grow" buttons. See it in action:
+
+<iframe width="960" height="720" src="https://www.youtube.com/embed/M43yTrJ0jzc?rel=0" frameborder="0" allowfullscreen></iframe>
+
+### Add new Hardware Information page
+
+On the System page the "Hardware" name is now a link to a new "Hardware
+information" page. For now this shows information about the system name, form
+factor, CPU, BIOS, and PCI devices. Other types of hardware will be added in
+future versions.
+
+![System Hardware Info](/images/system-hwinfo.png)
+
+### Load Application metadata (Appstream) packages on demand on Debian/Ubuntu
+
+[Cockpit 160](http://cockpit-project.org/blog/cockpit-160.html) started to
+install the [AppStream metadata](https://www.freedesktop.org/wiki/Distributions/AppStream/)
+on demand on the Apps page on Fedora. Debian, Ubuntu, and derivatives use a
+different mechanism for that, which is now being handled by Cockpit.
```

Comment:
Still no real applications listed here on Debian/Ubuntu right? If so, is this even worth mentioning? 

Until we have real stuff listed, this is a non-feature from a user point of view. Please correct me if I'm wrong and we have real usable applications listed.

Reply:
You are right. sosreport isn't a thing on Debian and Fleet commander etc. neither. So I removed it.